### PR TITLE
UI alignments

### PR DESCRIPTION
**Pull request by NetaMolcho**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -10,7 +10,8 @@
       loader="spinner"
     ></loading>
     <div>
-      <v-card-title>{{messages.title}}</v-card-title>
+      <v-card-title style="margin-bottom:16px;font-size:32px">{{messages.title}}</v-card-title>
```
Comment (tomer-epstein) [inline frontend/src/App.vue:13]: why inline?

File: frontend/src/components/Collections.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <v-row>
       <v-col cols="12">
         <div v-for="(collection, index) in collections" :key="index"><br>
-          <v-card-title class="prompt-title" style="font-size:16px">{{collection.title}}</v-card-title>
+          <v-card-title class="prompt-title" style="margin-bottom:8px;font-size:20px">{{collection.title}}</v-card-title>
```
Comment (tomer-epstein) [inline frontend/src/components/Collections.vue:18]: why inline style? 
we have the div.prompt-title class in globalStyles.css

**Review by tomer-epstein — COMMENTED** [review]
Looks good!
We want to merge Eyal PR first, the merge this PR.

File: frontend/src/components/Collections.vue (hunk 2 above)
Comment (NetaMolcho) [reply frontend/src/components/Collections.vue]: done

File: frontend/src/App.vue (hunk 1 above)
Comment (NetaMolcho) [reply frontend/src/App.vue]: done

**Review by tomer-epstein — APPROVED** [review]

**Pull request merged**